### DISCUSSION TITLE
[BUG] Not Working

### DISCUSSION THREAD

**Discussion by IzimiMakai** [ErinaSugino/Starbound-SBR-Mod#34] [Trouble Shooting] [answered]
I'm trying to do the Aphrodite's bow quest line, however, when I lay down on the node and the NPC joins me nothing happens, they just stay idle and I can't do anything except get out of the node. The position select screen doesn't appear either.

Log File:
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/11872558/starbound.log)

**Comment by ErinaSugino** [comment]
https://github.com/ErinaSugino/Starbound-SBR-Mod/wiki/Advanced-Troubleshooting

**Reply by IzimiMakai** [reply]
Already tried everything from there

**Reply by ErinaSugino** [reply, marked as answer]
No you didn't. You have POV Redux 4.0.1.
It clear states, on the main mod page, the troubleshooting guide and probably even POV Redux's mod page that only V4.2.0 is compatible with SBR.

**Reply by IzimiMakai** [reply]
LOL, sorry for being a fuckincg dumbass and taking so much of your time.
